types/obelisk.js/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for obelisk.js v1.1.0
+// Type definitions for obelisk.js v1.2.1
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `// Type definitions for obelisk.js 1.2`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!